tests/utils.js

```diff
@@ -1338,7 +1340,7 @@ module.exports = {
   tearDownServices: stopServices,
   endSession: async (exitCode) => {
     await module.exports.tearDownServices();
-    return module.exports.reporter.afterLaunch(exitCode);
+    await new Promise((resolve) => module.exports.reporter.afterLaunch(resolve.bind(this, exitCode)));
```

Comment:
This was the root issue, predating but perhaps exacerbated by, the retries change - the screenshot reporter afterLaunch function expects a callback to be passed.